nautobot/core/views/generic.py

```diff
@@ -1431,6 +1438,7 @@ class BulkComponentCreateView(GetReturnURLMixin, ObjectPermissionRequiredMixin,
 
     parent_model = None
     parent_field = None
+    primary_pattern_field = "name"
```

Comment:
I must not have committed those changes. I'm still in the middle of getting view tests and permissions working. This was to support ModuleBay using `position` instead of `name`